### PR TITLE
Fixed selectors for some class changes that Github made

### PR DESCRIPTION
**Pull request by csudcy**
GitHub decided to change a CSS class (again!) so I've updated the script.
Tested pulls, commits and compares with Tampermonkey on Chrome. Updated the version and changelog. Still waiting for GitHub to adopt as a builtin feature :smile:

### CODE REVIEW

File: ghAssistant.user.js [hunk 1]
```diff
@@ -453,7 +453,7 @@ gha.DomWriter.attachPerDiffFileFeatures = function () {
 };
 
 gha.DomWriter.makeFileNameKeyboardAccessible = function (child) {
-    var fileNameSpan = child.querySelector('.file-info > .js-selectable-text');
+    var fileNameSpan = child.querySelector('.file-info > .user-select-contain');
```
Comment (bbrandt) [inline ghAssistant.user.js]: This works great for me.  Thanks for the fix.

**Comment by bbrandt** [comment]
The CI failure seems to be an unrelated issue where the OK button is not collapsing the reviewed section.  This was failing before the pull request [in master](https://travis-ci.org/jakub-g/gh-code-review-assistant/builds/55666843).

**Comment by jakub-g** [comment]
Merged and propagated to master, thanks!

**Comment by csudcy** [comment]
Sweet, cheers :+1:

**Pull request merged**